README.md

```diff
@@ -8,24 +8,24 @@ The path follows the pattern: ``clients/{language}/github.com/{owner}/{repositor
 The ``{language}`` component of the path is the path-safe representation
 of the full language name which is mapped in [languages.json](./languages.json).
 
-Each client's JSON object represents the details displayed on the [clients documentation page](https://redis.io/docs/clients).
+Each client's JSON object represents the details displayed on the [clients documentation page](https://placeholderkv.io/docs/clients).
 
-For example [clients/python/github.com/redis](./clients/python/github.com/redis/redis-py.json):
+For example [clients/python/github.com/placeholderkv](./clients/python/github.com/placeholderkv/placeholderkv-py.json):
```

Comment:
Ok, so this changes a link to a file within the repo about the client named `redis-py`, to an invalid link. Just so that we're aware.